modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaPlayFrameworkCodegen.java

```diff
@@ -153,6 +153,9 @@ public void processOpts() {
         supportingFiles.add(new SupportingFile("routes.mustache", "conf", "routes"));
 
         //App/Utils folder
+        if (!this.controllerOnly && this.useInterfaces) {
```

Comment:
Now that I've read the code in more detail, I think what I am asking is what `controllerOnly` should do: emit the controllers, emit the interface, but don't emit an implementation.